sitzverteilung-frontend/src/components/result/ResultTable.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <v-dialog
+    v-model="dialog"
+    max-width="600px"
+  >
+    <v-card>
+      <v-card-title>{{ detailTitle }}</v-card-title>
+      <v-card-text>
+        <p>Detailinformationen {{ detailInfo }}</p>
+      </v-card-text>
+      <v-card-actions>
+        <v-btn
+          variant="text"
+          @click="dialog = false"
+          >Schließen</v-btn
+        >
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+  <v-toolbar flat>
+    <v-toolbar-title>Detailansicht zu:</v-toolbar-title>
+    <v-btn
+      variant="outlined"
+      class="mx-2"
+      @click="goToDetail('Hare/Niemeyer')"
+      >Hare/Niemeyer</v-btn
+    >
+    <v-btn
+      variant="outlined"
+      class="mx-2"
+      @click="goToDetail('Sainte-Laguë/Schepers')"
+      >Sainte-Laguë/Schepers</v-btn
+    >
+    <v-btn
+      variant="outlined"
+      class="mx-2"
+      @click="goToDetail('D`Hondt')"
+      >D'Hondt</v-btn
+    >
+  </v-toolbar>
+  <v-data-table
+    :headers="headers"
+    :items="results"
+    hide-default-footer
+    density="compact"
+    no-data-text="Keine Berechnungsdaten vorhanden."
+    items-per-page="-1"
```

Comment:
die -1 wird aktuell wie ein string behandelt. wenn es eine number sein soll, dann muss noch ein doppelpunkt davor


```suggestion
    :items-per-page="-1"
```